### ISSUE TITLE
exec shell doesn't work nicely with up/down arrow keys

### ISSUE THREAD

**Issue by ianneub** [tutumcloud/tutum-cli#62]
I'm having a small, but annoying issue with the `exec` command, and I'm not sure if it's my fault or something that could be fixed with the cli:

```
$ tutum exec 534ca3f1
# whoami
root
# bash
root@web-1:/var/www/current# whoami
root
root@web-1:/var/www/current# 
```
At this point I press the `up arrow` on my keyboard. Nothing is shown until I press `return` and then it looks as if everything was ok:
```
root@web-1:/var/www/current# whoami
root
```

Note: If i press the `up arrow` key twice it will show the `whoami` line, but when I hit `return` it will actually execute not the `whoami` command, but whatever was execute prior to that.

I'm on OS X 10.10.3, using iTerm 2.1.1. I installed the tutum cli 0.15.0.1 via homebrew.

**Comment by ianneub** [comment]
Here is a quick example:
https://www.youtube.com/watch?v=Clgb8wFJOn0&feature=youtu.be

**Comment by tifayuki** [comment]
fixed in 0.15.1
